.github/workflows/deploy.yml

```diff
@@ -174,7 +174,8 @@ jobs:
           gcr-token: ${{ secrets.GCR_TOKEN }}
       - run: |
           gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
-          ./scripts/deploy.sh -s ${target_stage} -t ${target_version}
+          ./scripts/deploy.sh -g -t ${target_stage} -v ${target_version} -r "${{ github.event.inputs.rfc }}"
```

Comment:
`-g` is a debug mode, more verbose.
also, had the incorrect args passed to `deploy.sh`